.yaspeller.json

```diff
@@ -4,6 +4,7 @@
   "maxRequests": 5,
   "ignoreDigits": true,
   "ignoreText": [["^layout:.*$", "m"]],
+  "ignoreTags": [["**", "__"]],
```

Comment:
Per https://github.com/hcodes/yaspeller readme, `ignoreText` and `ignoreTags` is supposed to be Array, not array of Arrays.

Reply:
Ok then